### ISSUE TITLE
FR: Benchmark common (and uncommon) configurations

### ISSUE THREAD

**Issue by cceckman** [marcusolsson/tui-go#88]
#14 notes some performance issues with re-rendering `Label`s (since word wrapping is a nontrivial CPU operation).

It would be useful to have some benchmarks that could call out when a chance has a significant impact on performance.

**Comment by marcusolsson** [comment]
Agreed. I just fired up pprof back then but it makes sense to benchmark operations like `Draw` and `SizeHint`.
